### ISSUE TITLE
Error #3000

### ISSUE THREAD

**Issue by VeritasDev** [Chris-Perkins/TwitchPlayer#3]
How do you get around the failure to play the video due to error #3000? I can't get twitch to play using TwitchPlayer or even in safari..

**Comment by Chris-Perkins** [comment]
Hi,
Please see this page by twitch on how to fix this error: https://streamdiag.com/fix-twitch-error-3000/

**Comment by VeritasDev** [comment]
All of those errors seem to be related to experiencing that error in a browser, rather than on an embedded stream.

**Comment by Chris-Perkins** [comment]
The same resources are used for embedded players and players you see on Twitch, so by clearing your cookies you will likely resolve this issue. You can clear your cookies using the following function (requires WebKit import):

```Swift
    /// Call this to clear the cache of WKWebViews.
    ///
    /// - Parameter completionHandler: The completion handler for the completion of wkwebview cache removal.
    public static func clearWKWebViewCache(completionHandler: @escaping () -> Void) {
        if #available(iOS 9.0, *) {
            let websiteDataTypes = NSSet(array: [WKWebsiteDataTypeCookies])
            let date = NSDate(timeIntervalSince1970: 0)
            WKWebsiteDataStore.default().removeData(ofTypes: websiteDataTypes as! Set<String>,
                                                    modifiedSince: date as Date, completionHandler: completionHandler)
        } else {
            var libraryPath = NSSearchPathForDirectoriesInDomains(FileManager.SearchPathDirectory.libraryDirectory, FileManager.SearchPathDomainMask.userDomainMask, false).first!
            libraryPath += "/Cookies"

            do {
                try FileManager.default.removeItem(atPath: libraryPath)
            } catch {
                print("error")
            }
            URLCache.shared.removeAllCachedResponses()
        }
    }
```

Please try running the function and let it complete its thread before displaying the twitch player; this should resolve the issue. If it does not, I'll need to gather some more info.

**Comment by VeritasDev** [comment]
Great that did it! TYVM!
